test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHelpers.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Linq;
+using MsgPack;
+
+namespace Microsoft.AspNetCore.SignalR.Common.Tests.Internal.Protocol
+{
+    public static class MessagePackHelpers
+    {
+        public static MessagePackObject Arr(params MessagePackObject[] items) =>
```

Comment:
Array?

Reply:
Yeah, I was toeing the line between descriptive and not having long theory data lines. I could probably expand this though, it doesn't end up as common as I thought it would.